cmd/create/project.go

```diff
@@ -13,15 +13,15 @@ import (
 )
 
 const (
-	projectShort = "Create project resources"
+	projectShort = "Creates project resources"
 	projectLong  = `
-Create the projects.(project,projects can be used interchangeably in these commands)
+Create the projects.(project/projects can be used interchangeably in these commands)
 
 ::
 
  bin/flytectl create project --name flytesnacks --id flytesnacks --description "flytesnacks description"  --labels app=flyte
 
-Create Project by definition file. Note: The name shouldn't contain any whitespace characters'
+Creates Project by definition file. Note: The name shouldn't contain any whitespace characters'
 ::
 
  bin/flytectl create project --file project.yaml 
```

Comment:
can we replace `bin/flytectl` with `flytectl`?

Reply:
Yes, Will do that